nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -60,6 +60,14 @@
 except (ImportError, ModuleNotFoundError):
     HAVE_APEX = False
 
+try:
+    import transformer_engine
```

Comment:
Is TE painful to install like apex or can we make it a NeMo NLP requirement if pip installable?

Reply:
I actually tried to follow the instructions from their github in my conda env and the install failed :( I've pinged kirthi about this but haven't heard back yet. This is why I haven't updated the README yet

Reply:
Cool does this mean we won't need import guards eventually?